core/src/apps/benfen/tx_parser.py

```
@@ -0,0 +1,432 @@
+                    "TransactionKind": {
+                        "ProgrammableTransaction": {
+                            "Inputs": formatted_inputs,
+                            "Commands": result["commands"],
+                            "Sender": {"Address": result["sender"]},
+                        }
+                    },
+                    "GasData": result["gas_data"],
+                    "Expiration": result["expiration"],
+                }
+            }
+            return final_result
```

Ruff (0.8.2)

Consider moving this statement to an `else` block

(TRY300)